domain/types.ts

```diff
@@ -53,6 +53,11 @@ export type TransactionCategories = {
     category: string
 };
 
+export type AccountStatement = {
```

Comment:
You'll need to move these types to `/integrations/stitch/types.ts` sorry about that. Not really any domain types being used -- just using the Stitch types